var/spack/repos/builtin/packages/nwchem/package.py

```diff
@@ -35,6 +37,8 @@ class Nwchem(Package):
 
     variant("openmp", default=False, description="Enables OpenMP support")
     variant("mpipr", default=False, description="Enables ARMCI with progress rank")
+    variant("armcimpi", default=False, description="Enables ARMCI-MPI")
```

Comment:
AFAICT from the documentation `ARMCI_NETWORK=MPI-PR` and `ARMCI_NETWORK=ARMCI` are mutually exclusive so `+mpipr` and `+armcimpi` should be a single multi-valued variant.

Reply:
i don't know how to implement that.  i guess i'll figure it out.

Reply:
Thanks!

Reply:
see 642958c6759e1cfe62866ed0e133fd045dae7df0